vesin/torch/src/vesin_torch.cpp

```diff
@@ -19,6 +22,8 @@ static VesinDevice torch_to_vesin_device(torch::Device device) {
 static torch::Device vesin_to_torch_device(VesinDevice device) {
     if (device == VesinCPU) {
         return torch::Device("cpu");
+    } else if (device == VesinCUDA) {
+        return torch::Device("cuda");
```

Comment:
This device is then used together with `torch::from_blob` to wrap vesin pointers inside torch tensors, meaning it should be created with the correct id. 

We can either get the id from the input tensors and pass it as another parameter, adding a device id inside VesinDevice (which I would prefer).